lib/engine/config/tile.rb

```diff
@@ -30,6 +30,7 @@ module Tile
         '56' => 'town=revenue:10;town=revenue:10;path=a:0,b:_0;path=a:_0,b:2;path=a:1,b:_1;path=a:_1,b:3',
         '57' => 'city=revenue:20;path=a:0,b:_0;path=a:_0,b:3',
         '58' => 'town=revenue:10;path=a:0,b:_0;path=a:_0,b:2',
+        '59s' => 'city=revenue:20;path=a:0,b:_0',
```

Comment:
Game-specific tiles should not be put into this file. They should go into game/g_1829/map.rb

Reply:
moved to g_1829/map.rb